.gitignore

```diff
@@ -90,9 +90,16 @@ doc/gapmacrodoc.idx
 /libgap.la
 .libs/
 
+/tst/testlibgap/api
 /tst/testlibgap/basic
-/tst/testlibgap/basic.out
 /tst/testlibgap/wscreate
-/tst/testlibgap/wscreate.out
 /tst/testlibgap/wsload
-/tst/testlibgap/wsload.out
+/tst/testlibgap/*.out
+
+/tst/testkernel/dstruct
+/tst/testkernel/*.out
+
+# emacs backup and lock files
+*~
+\#*\#
+.\#*
```

Comment:
Huh... I think you ought to rebase on lastest master now, you are several months behind.

Reply:
Huh, I thought I had copied that from a freshly pulled master, because I was getting merge issues (which is what the last commit is all about). Weird, I'll have to look at it again.

Reply:
This stuff is definitely in master. But it is not in the old HPC-GAP PR on which you based this PR; that one was last updated in June. So a `git rebase master` should get rid of the above.